fhd_core/beam_modeling/fhd_struct_init_antenna.pro

```diff
@@ -4,23 +4,19 @@ FUNCTION fhd_struct_init_antenna,obs,beam_model_version=beam_model_version,$
     psf_dim=psf_dim,psf_max_dim=psf_max_dim,beam_offset_time=beam_offset_time,debug_dim=debug_dim,$
     inst_tile_ptr=inst_tile_ptr,ra_arr=ra_arr,dec_arr=dec_arr,fractional_size=fractional_size,$
     kernel_window=kernel_window,beam_per_baseline=beam_per_baseline,$
-    beam_gaussian_decomp=beam_gaussian_decomp,conserve_memory=conserve_memory,_Extra=extra
+    beam_gaussian_decomp=beam_gaussian_decomp,conserve_memory=conserve_memory,$
+    import_pyuvdata_beam_filepath=import_pyuvdata_beam_filepath,$
+    use_psf_resolution=use_psf_resolution,_Extra=extra
+
 t0=Systime(1)
 
 IF keyword_set(conserve_memory) then begin
   IF conserve_memory GT 1E6 THEN mem_thresh=conserve_memory ELSE mem_thresh=1E8 ;in bytes
 ENDIF
 
 IF N_Elements(beam_model_version) EQ 0 THEN beam_model_version=1
+IF N_Elements(use_psf_resolution) GT 0 THEN psf_resolution=use_psf_resolution
 instrument=obs.instrument
-tile_gain_fn=instrument+'_beam_setup_gain' ;mwa_beam_setup_gain
-tile_init_fn=instrument+'_beam_setup_init' ;mwa_beam_setup_init
-
-;If other phases of the mwa are used, use the proper gain and init functions
```

Comment:
Should keep this logic regarding other mwa phases (specifically phase II).

Reply:
This was just moved to lines 81-82